lib/core/form/services/activiti-alfresco.service.ts

```diff
@@ -54,15 +54,12 @@ export class ActivitiContentService {
 
     /**
      * Returns a list of all the repositories configured
-     *
-     * @param accountId
-     * @param folderId
+     * @param includeAccounts
      */
-    getAlfrescoRepositories(tenantId: number, includeAccount: boolean): Observable<any> {
+    getAlfrescoRepositories(includeAccounts: boolean): Observable<any> {
```

Comment:
tenantId is missing

Reply:
I'm not sure if is needed. In all places that we use this function we pass tenantId = null. The api is returning the same if we don't pass anything as tenantId or if we pass only the one that the user is part of (if he is part of any tenant). The only place that tenantId could be useful is when you are an admin user and you can query by using tenantId but I think this is not a valid case in our apps.

Reply:
I can re-add it it's just a thought. Thnx @eromano

Reply:
it was not used. the UI was passing null